src/css/controls/imports/slider.less

```diff
@@ -120,6 +120,10 @@
     align-items: center;
     background: transparent none;
     padding: 0 12px;
+}
+
+.jw-slider-time,
+.jw-horizontal-volume-container {
```

Comment:
@pajong `jw-horizontal-volume-container` will never contain cues or a buffer bar right? I'm wondering how much additional css does this produce and couldn't we keep buffer and cue rules outside of vertical and volume sliders?